src/YNCursesUI.cc

```diff
@@ -423,3 +424,24 @@ void YNCursesUI::setConsoleFont( const std::string & console_magic,
     }
 }
 
+
+YWidget * YNCursesUI::askSendWidgetID()
+{
+    std::string id = NCPopupTextEntry::askForText( wpos( 0, 0 ),
+                                                   "Widget ID:", // label
```

Comment:
NP: The label should be consistent with Qt which uses "Enter Widget ID:" label (https://github.com/libyui/libyui-qt/pull/74/files/bb0fa0426b545c8740007bba8c14a7d1cfbd91d9#diff-0ffb2fa84e56876f016657d53238482eR625)

Plus should be marked for translation.